regolith/helpers/a_expensehelper.py

```diff
@@ -0,0 +1,176 @@
+"""
+Helper to add expenses.
+"""
+import datetime as dt
+import dateutil.parser as date_parser
+from dateutil.relativedelta import relativedelta
+import sys
+
+from regolith.helpers.basehelper import DbHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+)
+
+TARGET_COLL = "projecta"
+ALLOWED_TYPES = ["business", "travel"] # need to check all expense types.
+ALLOWED_STATI = ["submitted", "unsubmitted", "reimbursed"]
+
+
+def subparser(subpi):
+    subpi.add_argument("amount", help="expense amount",
+                       )
+    subpi.add_argument("name", help="A short name for the expense",
+                       default=None
+                       )
+    subpi.add_argument("purpose", help="A short description of expense purpose",
+                       default=None)
+
+    subpi.add_argument("-b", "--business", action='store_true',
+                       help="expense type is business. If not specified defaults to travel"
+                       )
+    subpi.add_argument("-y", "--payee",
+                       help="payee of the expense. defaults to sbillinge",
+                       default="sbillinge"
+                       )
+    subpi.add_argument("-g", "--grants", nargs="+",
+                       help="grant, or list of grants that cover this expense. Defaults to tbd"
+                       )
+    subpi.add_argument("-s", "--status",
+                       help=f"status, from {ALLOWED_STATI}. Default is unsubmitted"
+                       )
+    subpi.add_argument("-z", "--segregated",
+                       help="Amount for any segregated expense. Defaults to 0",
+                       default="0"
```

Comment:
is it allowed to be an int or it has to be a string?

btw, I think I meant for you to take the next step after I hinted the first step, and to think about defaults for all the optional args where it makes sense.....

Reply:
Okay, I see now. I will fix this, before proceeding with the helper I am going to try, and make sure I have my test for travel expenses up.

Reply:
yes, perfect!  TDD.  You are getting it!  Thanks so much